web/assets/todays-date.js

```diff
@@ -0,0 +1,14 @@
+const todaysDate = () => {
+    const selectTodayLink = document.querySelector('[data-module="select-todays-date"]');
+    const datePicker = document.querySelector('.date-picker');
```

Comment:
This isn't necessary, but since you mentioned it, the way the GDS components work with data-module is something like:
```html
<div data-module="select-todays-date">
    <input type="date" />
    <a>Use today's date</a>
</div>
```

So that you can then do:
```suggestion
    const modules = document.querySelectorAll('[data-module="select-todays-date"]');
    Array.from(modules).forEach(module => {
        const selectTodayLink = module.querySelector('a');
        const datePicker = module.querySelector('input[type="date"]');
        ...
    });
```
That way we can easily support multiple instances of the date selector on the same page without worrying about them crossing over with each other.